### DISCUSSION TITLE
NiceGui with styling template such as Quasar layouts

### DISCUSSION THREAD

**Discussion by megarg** [zauberzeug/nicegui#3127] [Q&A]
### Question

NiceGUI is great but the default look-n-feel is not attractive. I have 2 questions which may really help serious projects being built with NiceGUI:
1. Is there sample available or easy to create application similar to Quasar layout gallery. Such as:
    - https://quasar.dev/layout/gallery/quasar-classic-dark/    
    - https://quasar.dev/layout/gallery/youtube/
    
2. If above is not possible, should we use something like "daisyui" because it may provide standard set of CSS classes then we can have a centralized CSS to manage it. Is there a sample/example to use with Daisyui or any other mechanism to make look-n-feel easy to manage across an enterprise class application (across multiple developers and multiple pages/functionalities).

I am new to NiceGUI. We are seriously considering building our application using NiceGUI but I am finding look-n-feel aspect too hard and time consuming for me to understand. I am primarily java and python developer so not very good at CSS.

**Comment by rodja** [comment]
The demos you have linked simply use the standard mechanisms of styling which are also supported by NiceGUI. For sharing a common theme I suggest you look at https://github.com/zauberzeug/nicegui/tree/main/examples/modularization.
